modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -787,25 +787,34 @@ See xref:configuration/files/encryption_configuration.adoc[encryption configurat
 
 `encryption:recreate-master-key` decrypts the ownCloud file system, replaces the existing master key with a new one, and encrypts the entire ownCloud file system with the new master key. Given the size of your ownCloud filesystem, this may take some time to complete. However, if your filesystem is quite small, then it will complete quite quickly. The `-y` switch can be supplied to automate acceptance of user input.
 
-[[federation-sync]]
-== Federation Sync
+== Federated Shares
 
-Synchronize the address books of all federated ownCloud servers.
+=== Synchronize All Federated Server Address Books
+
+Servers connected with federation shares can share user address books and auto-complete usernames in share dialogs. 
+Use this command to synchronize federated servers:
 
 [source,console]
 ----
-federation:sync-addressbooks  Synchronizes address books of all federated clouds
+sudo -u www-data ./occ federation:sync-addressbooks
 ----
 
-Servers connected with federation shares can share user address books,
-and auto-complete usernames in share dialogs. Use this command to
-synchronize federated servers:
+NOTE: This command is only available when the "Federation" app (`federation`) is enabled.
 
-....
-sudo -u www-data php occ federation:sync-addressbooks
-....
+=== Poll Incoming Federated Shares For Updates
 
-NOTE: This command is only available when the "Federation" app (`federation`) is enabled.
+For performance reasons, federated shares do not update automatically.
+Instead, federated share directories are only updated when users browse them using web UI.
+ownCloud and system administrators can use the `incoming-shares:poll` to poll federated shares for updates.
+The command polls all received federated shares. It does not require a path.
+
+[source,console]
+----
+sudo -u www-data ./occ incoming-shares:poll
```

Comment:
We should use `sudo -u www-data php occ`, same for above.
(Might be that there are some leftovers at other places too, but that could be corrected in an other PR)